alignak/objects/config.py

```diff
@@ -1103,13 +1103,14 @@ def read_config_buf(self, buf):  # pylint: disable=R0912
 
             line_nb += 1
             # Remove comments
-            line = split_semicolon(line)[0].strip()
+            if not continuation_line:
+                line = split_semicolon(line)[0].strip()
 
             # A backslash means, there is more to come
             if re.search(r"\\\s*$", line) is not None:
                 continuation_line = True
                 line = re.sub(r"\\\s*$", "", line)
-                line = re.sub(r"^\s+", " ", line)
+                # line = re.sub(r"^\s+", " ", line)
```

Comment:
Why comment this? We are replacing line by only one space. This affect all file parsing here, carful